atomic_reactor/plugins/add_dockerfile.py

```diff
@@ -110,6 +110,20 @@ def add_dockerfile(self, build_dir: BuildDir) -> None:
 
         self.log.info("added %s for the %s build", self.df_path, build_dir.platform)
 
+    def _modify_dockerignore(self, build_dir: BuildDir) -> None:
+        dockerignore = build_dir.path / DOCKERIGNORE
+
+        if not dockerignore.exists():
+            return
+
+        with open(dockerignore, "a") as f:
+            f.write('\n!/Dockerfile*\n')
```

Comment:
Shouldn't we say explicitly which Dockerfile and json ?

Reply:
I don't think it matters that much, since we are allowing them only in /

either way, even if we would allow just specific ones, AND they would do COPY /*   they would copy those anyway

so if they are relying on this for such copy command, they will be already copying our files anyway

Reply:
is it hard to use exacts filenames OSBS, or why we have to do just a semi-fix that may result in another issue in future when somebody will need to avoid copying some json files?

Reply:
I agree that we should explicitly add files that are being ADDed in the Dockerfile. Also, I am not sure that we should limit this to .json files and Dockerfiles

Reply:
ok then, I'll change it